guide/14-deep-learning/how-sequencetosequence-works.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<ol><li>Add space between length and ( in the sentence <strong><u>output sequence of any length(independent</u></strong></li><li><s>Few such tasks are: </s> Sequence translation includes tasks such as machine translation, summarization, question answering, etc. (include relevant links from above)</li><li>Provide image description similar to - &lt;center&gt;Figure1: High level view of a transformer&lt;/a&gt;&lt;/center&gt;</li><li><s>Transformers library gives</s> Either use <strong>Hugging Face Transformers library provides</strong> or <strong>This library provides</strong></li><li><s>consists of an Encoder block and a decoder block</s> - consists of an <strong>encoder block</strong> and a <strong>decoder block</strong></li><li><s>Many of the latest transformer-based architectures like Bert, RoBerta, AlBert utilizes</s> - Many of the latest transformer  based architectures like BERT, RoBERTa, ALBERT utilizes</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/869/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>fixed</p>